src/drivers/intel/hda/hda-dma.c

```diff
@@ -28,6 +28,68 @@
 #include <stddef.h>
 #include <stdint.h>
 
+/*
+ * HD-Audio DMA programming sequence
+ *
+ * To start a stream (Playback):
+ * 1. Host sends the PCM_PARAMS IPC with two_step_stop flag set
+ * 2. Host sets RUN bit for link DMA
+ * 3. Host sets RUN bit for host DMA
+ * 4. Host sends the STREAM_TRIG_START IPC to the DSP
+ * 5. FW starts the pipeline and sets GEN bit to 1
+ *
+ * To start a stream (Capture):
+ * 1. Host sends the PCM_PARAMS IPC with two_step_stop flag set
+ * 2. Host sets RUN bit for host DMA
+ * 3. Host sends the STREAM_TRIG_START IPC to the DSP
+ * 4. FW starts the pipeline and sets GEN bit to 1
+ * 5. Host sets RUN bit for link DMA
+ *
+ * To pause an active stream (Playback):
```

Comment:
instead of 'to pause' I would use 'PAUSE_PUSH on an active stream (playback)'

pause/release can be overloaded terms, the ALSA definitions are less ambiguous.